homeassistant/components/myuplink/quality_scale.yaml

```diff
@@ -0,0 +1,99 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      No custom actions are defined.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
In the `test_full_flow` we should also assert the data of the newly created config entry and the unique_id